### PR TITLE
Cfy 4726 bootstrap redo

### PR DESCRIPTION
**Pull request by nir0s**

### CODE REVIEW

File: content/manager/bootstrapping.md [hunk 1]
```diff
@@ -41,31 +45,65 @@ Please verify the [prerequisites]({{< relref "manager/bootstrapping.md" >}}) bef
...
 $ mkdir -p ~/cloudify-manager
 $ cd ~/cloudify-manager
-$ git clone https://github.com/cloudify-cosmo/cloudify-manager-blueprints
-$ cd cloudify-manager-blueprints
-$ git checkout -b cloudify <tag>
+$ curl -L https://github.com/cloudify-cosmo/cloudify-manager-blueprints/archive/3.3.1.tar.gz -o cloudify-manager-blueprints.tar.gz
+$ tar -xzvf cloudify-manager-blueprints.tar.gz
+$ cd cloudify-manager-blueprints-3.3.1
+$ ls -l
 ...
```
Comment (EarthmanT) [inline content/manager/bootstrapping.md]: why are you changing this

File: content/manager/bootstrapping.md [hunk 2]
```diff
@@ -41,31 +45,65 @@ Please verify the [prerequisites]({{< relref "manager/bootstrapping.md" >}}) bef
...
+-rw-r--r--  1 nir0s users  27K Jan  4 13:32 aws-ec2-manager-blueprint.yaml
+-rw-r--r--  1 nir0s users  15K Jan  4 13:32 openstack-manager-blueprint-inputs.yaml
+-rw-r--r--  1 nir0s users  29K Jan  4 13:32 openstack-manager-blueprint.yaml
+-rw-r--r--  1 nir0s users  13K Jan  4 13:32 simple-manager-blueprint-inputs.yaml
+-rw-r--r--  1 nir0s users  23K Jan  4 13:32 simple-manager-blueprint.yaml
+-rw-r--r--  1 nir0s users  14K Jan  4 13:32 vcloud-manager-blueprint-inputs.yaml
+-rw-r--r--  1 nir0s users  32K Jan  4 13:32 vcloud-manager-blueprint.yaml
+-rw-r--r--  1 nir0s users  16K Jan  4 13:32 vsphere-manager-blueprint-inputs.yaml
+-rw-r--r--  1 nir0s users  29K Jan  4 13:32 vsphere-manager-blueprint.yaml
+...
```
Comment (EarthmanT) [inline content/manager/bootstrapping.md]: Picky, but I don't think we should all see your username.

File: content/manager/bootstrapping.md [hunk 3]
```diff
@@ -41,31 +45,65 @@ Please verify the [prerequisites]({{< relref "manager/bootstrapping.md" >}}) bef
...
+As you can see there are manager blueprints for different environments. Each of these blueprints provisions resources (like a vm, security groups, ip's, etc..) required for the manager to run and installs the different components on the vm.
 
 Now let's move on to configuration.
 
-Each manager blueprint has an `inputs` file associated with it. That input file is the configuration for the environment the manager should be bootstrapped on and general configuration for the manager itself.
+Each manager blueprint has an `inputs.yaml` file associated with it. That inputs file is the configuration for the environment the manager should be bootstrapped on and general configuration for the manager itself.
 
 While defaults are provided for most inputs, some inputs are required since they relate to your specific provider's account like credentials for example.
 
+The inputs file contains a description of each input. You should modify the inputs file to contain the relevant information for your environment.
```
Comment (EarthmanT) [inline content/manager/bootstrapping.md]: Maybe we should point at this stage to the IaaS Manager pages?

Comment (nir0s) [inline content/manager/bootstrapping.md]: Yeah... but we don't have them yet.

File: content/manager/bootstrapping.md (hunk 2 above)
Comment (nir0s) [inline content/manager/bootstrapping.md]: good point.

File: content/manager/bootstrapping.md (hunk 1 above)
Comment (nir0s) [inline content/manager/bootstrapping.md]: true. i'll vary between the two.

Comment (nir0s) [inline content/manager/bootstrapping.md]: I'm not willing to have people install git to bootstrap. it is ridiculous. I'll have them download a tar/zip depending on the system

**Pull request merged**